CONTRIBUTING.md

```diff
@@ -1,15 +1,64 @@
 # Contributing
 
+🎈 Thanks for your help improving the project! We are so happy to have you!
+
 🚨 Before making any non-trivial change, please first open an issue describing the change to solicit feedback and guidance. This will increase the likelihood of the PR getting merged.
 
-Please note we have a code of conduct, please follow it in all your interactions with the project.
+In general, the smaller the diff the easier it will be for us to review quickly.
+
+Please note we have a [Code of Conduct](#code-of-conduct), please follow it in all your interactions with the project.
+
+# Pull Request Process
 
-## Pull Request Process
+## Steps for the PR author
 
 1. Ensure that tests pass and code is lint free. You can run `yarn test` and `yarn lint` locally to check.
-2. Update the README.md if any changes invalidate its current content.
-3. Include tests for any new functionality.
-4. Reference relevant issues in your PR comment.
+1. Update the README.md if your changes invalidate or extend its current content.
+1. Include tests for any new functionality.
+1. Ensure each section of the [Pull Request Template](./PULL_REQUEST_TEMPLATE.md) is filled out.
```

Comment:
One additional thing that surfaced today was to explicitly delete the boilerplate sections if they do not apply to the PR

Reply:
Thx, I addressed that here and the [comment](https://github.com/ethereum-optimism/.github/pull/4/files#diff-479755504b0aad0fed26047714591511b4d07c76bc8521171b87a5f16100f850R2) in the PR template